docs/pages/kubernetes-access/guides/standalone-teleport.mdx

```diff
@@ -16,15 +16,20 @@ cluster you want to register. You can use `kubectl config get-contexts` to
 verify that the correct cluster is selected, or `kubectl config use-context
 ${CONTEXT_NAME?}` to switch to cluster `CONTEXT_NAME`.
 
-Next, use [get-kubeconfig.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh) to create a `kubeconfig` for Teleport to use.
+Next, use [get-kubeconfig.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh) to create a `kubeconfig` for Teleport to use. This will create the `teleport` namespace, the `teleport-sa` serviceaccount, a `teleport-role` clusterrole, and a corresponding `teleport-crb` clusterrolebinding.
```

Comment:
I think this would be clearer as a list since it's a fairly dense collection of code-style text. Also suggested using plain-English versions of Kubernetes terms.

```suggestion
Next, use [get-kubeconfig.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh) to create a `kubeconfig` for Teleport to use. This will create:
- The `teleport` namespace
- The `teleport-sa` service account
- The `teleport-role` cluster role
- The `teleport-crb` cluster role binding
```